src/components/src/ProductDisplayItem/productdisplayitem.main.constants.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright © 2019 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+// eslint-disable-next-line import/prefer-default-export
+export const REQUISITION_LISTS_ZOOM : string[] = [
```

Comment:
Are we planning to split out all component zooms like this? Just thinking as we eventually use the request library, will this be additional cleanup for later.

Reply:
True, might be unecessary work.  @dusanradovanovic will this be wasted work here?  do we still need to form zoom calls for the library?